forum/frontend/src/components/StartupsList.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import '../styles/startupsList.css'; 
+import { Link } from 'react-router-dom';
+
+const StartupsList = () => {
+  const [startups, setStartups] = useState([]);
+
+  useEffect(() => {
+    const accessToken = localStorage.getItem('accessToken'); 
+
+    console.log('Sending request to API with access_token:', accessToken);
+
+    axios.get('http://localhost:8000/api/startups/list/', {
+      headers: {
+        'Authorization': `Bearer ${accessToken}`  
+      }
+    })
+      .then(response => {
+        console.log('Response from API:', response.data);  
+        setStartups(response.data);
+      })
+      .catch(error => {
+        console.error('Error fetching startups:', error);  
+      });
+  }, []);
+
```

Comment:
If the list of startups is empty, the user sees nothing.  Add an empty state message to inform users when no startups are available.